src/dom/get-scroll-element.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Handle variations between browsers that are relevant when measuring the
+ * position of elements relative to one another.
+ */
+export function getScrollElement(): Element {
```

Comment:
Could this move to dom?  I think we were trying to reduce file count?

Reply:
Done.